### ISSUE TITLE
GenerateContributorsFile broken

### ISSUE THREAD

**Issue by subotic** [dhlab-basel/Knora#797]
@benjamingeer could you please take a look when you have time? The following error happens with and without a supplied token:

```
sbt:webapi> runMain org.knora.webapi.util.GenerateContributorsFile -t myprivatetoken
[info] Packaging /Users/subotic/_github.com/knora/webapi/target/scala-2.12/webapi_2.12-1.3.0.jar ...
[info] Done packaging.
[info] Running (fork) org.knora.webapi.util.GenerateContributorsFile -t bcb20132ec1ed3bbc2fbd674792a4b81aa4f4bb2
[error] objc[3238]: Class JavaLaunchHelper is implemented in both /Library/Java/JavaVirtualMachines/jdk1.8.0_112.jdk/Contents/Home/jre/bin/java (0x1038b84c0) and /Library/Java/JavaVirtualMachines/jdk1.8.0_112.jdk/Contents/Home/jre/lib/libinstrument.dylib (0x1039504e0). One of the two will be used. Which one is undefined.
[error] Exception in thread "main" java.lang.ClassCastException: spray.json.JsNull$ cannot be cast to spray.json.JsString
[error] 	at org.knora.webapi.util.GenerateContributorsFile$.$anonfun$contributorsWithNames$1(GenerateContributorsFile.scala:71)
[error] 	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:234)
[error] 	at scala.collection.Iterator.foreach(Iterator.scala:929)
[error] 	at scala.collection.Iterator.foreach$(Iterator.scala:929)
[error] 	at scala.collection.AbstractIterator.foreach(Iterator.scala:1417)
[error] 	at scala.collection.IterableLike.foreach(IterableLike.scala:71)
[error] 	at scala.collection.IterableLike.foreach$(IterableLike.scala:70)
[error] 	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
[error] 	at scala.collection.TraversableLike.map(TraversableLike.scala:234)
[error] 	at scala.collection.TraversableLike.map$(TraversableLike.scala:227)
[error] 	at scala.collection.AbstractTraversable.map(Traversable.scala:104)
[error] 	at org.knora.webapi.util.GenerateContributorsFile$.delayedEndpoint$org$knora$webapi$util$GenerateContributorsFile$1(GenerateContributorsFile.scala:66)
[error] 	at org.knora.webapi.util.GenerateContributorsFile$delayedInit$body.apply(GenerateContributorsFile.scala:35)
[error] 	at scala.Function0.apply$mcV$sp(Function0.scala:34)
[error] 	at scala.Function0.apply$mcV$sp$(Function0.scala:34)
[error] 	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
[error] 	at scala.App.$anonfun$main$1$adapted(App.scala:76)
[error] 	at scala.collection.immutable.List.foreach(List.scala:389)
[error] 	at scala.App.main(App.scala:76)
[error] 	at scala.App.main$(App.scala:74)
[error] 	at org.knora.webapi.util.GenerateContributorsFile$.main(GenerateContributorsFile.scala:35)
[error] 	at org.knora.webapi.util.GenerateContributorsFile.main(GenerateContributorsFile.scala)
[error] java.lang.RuntimeException: Nonzero exit code returned from runner: 1
[error] 	at sbt.ForkRun.processExitCode$1(Run.scala:33)
[error] 	at sbt.ForkRun.run(Run.scala:42)
[error] 	at sbt.Defaults$.$anonfun$bgRunMainTask$6(Defaults.scala:1147)
[error] 	at sbt.Defaults$.$anonfun$bgRunMainTask$6$adapted(Defaults.scala:1142)
[error] 	at sbt.internal.BackgroundThreadPool.$anonfun$run$1(DefaultBackgroundJobService.scala:366)
[error] 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
[error] 	at scala.util.Try$.apply(Try.scala:209)
[error] 	at sbt.internal.BackgroundThreadPool$BackgroundRunnable.run(DefaultBackgroundJobService.scala:289)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[error] 	at java.lang.Thread.run(Thread.java:745)
[error] (Compile / runMain) Nonzero exit code returned from runner: 1
[error] Total time: 7 s, completed Mar 28, 2018 10:15:02 AM
```

**Comment by benjamingeer** [comment]
Will do this today.

**Comment by subotic** [comment]
No stress. I'm just skipping this step :-)

**Comment by benjamingeer** [comment]
This was because a contributor can choose not to enter their name in their GitHub account. Fixed in #833.

**Comment by subotic** [comment]
Thanks :-)
